docs/src/pages/docs/[[...mdx]].js

```diff
@@ -114,11 +114,16 @@ export const getStaticPaths = async () => {
 };
 
 export const getStaticProps = async ({ params }) => {
-  const mdxDirFiles = getMDXDirFiles(params);
+  const mdxDir = getMDXDirFiles(params);
+
+  // We need to make sure this is only actual files and not a directory
+  // so we are filtering it to make sure the size of the file is not zero.
+  const mdxDirFiles = mdxDir.filter((file) => file.size > 0 && file);
```

Comment:
This is to make sure we are trying to parse MDX for actual MDX files and not a directory.